dali/pipeline/operators/optical_flow/optical_flow.h

```diff
@@ -65,7 +65,9 @@ class OpticalFlow : public Operator<Backend> {
           optical_flow_(std::unique_ptr<optical_flow::OpticalFlowAdapter<ComputeBackend>>(
                   new optical_flow::OpticalFlowStub<ComputeBackend>(of_params_))),
           image_type_(spec.GetArgument<decltype(this->image_type_)>(detail::kImageTypeArgName)),
-          device_id_(spec.GetArgument<int>("device_id")) {
+          device_id_(spec.GetArgument<int>("device_id")),
+          frames_width_(-1), frames_height_(-1), depth_(-1), nsequences_(-1), hints_width_(-1),
```

Comment:
how about doing that at the member variable definition instead?

Reply:
Done